### PR TITLE
feat: add user schema

### LINKED ISSUE #8: [FEATURE] <user id schema>
### ⭐ Suggestion

For registering or sign in, user data is required for verifying the later actions going to happen in web app. Therefore the user schema has to be made for required details.

### 💻 Use Cases

_No response_

### ❌ Related Problems

_No response_

### PR DESCRIPTION
**Pull request by chirag0002**
For registration user schema has to be made for storing user details for further future verification and actions.
It contains details such as:
- First Name
- Last Name
- User Name
- E-mail
- Phone No.
- Date Of Birth
- Country
- Password
- Confirm Password

Closes #8

### CODE REVIEW

File: server/Routes/Routes.js [hunk 1]
```diff
@@ -19,4 +22,32 @@ Router.post("/check", async (req, res) => {
   }
 });
 
+Router.post('/register', async (req,res) => {
+    console.log('inside register');
+
```
Comment (AayushShuk1a) [inline server/Routes/Routes.js:27]: Avoid pushing unnecessary consoles.

File: server/Routes/Routes.js [hunk 2]
```diff
@@ -19,4 +22,32 @@ Router.post("/check", async (req, res) => {
...
+        else{
+            const user = new User(userDetails);
+            await user.save();
+            res.status(201).json({message:"User Registered successfully"});
+        }
+    } catch(err){ 
+        console.log(err)
+    }
+});
+
```
Comment (AayushShuk1a) [inline server/Routes/Routes.js:52]: Try using comments for more user friendly experience.

**Review by AayushShuk1a — CHANGES_REQUESTED** [review]

**Review by chandankumar1307 — APPROVED** [review]
LGTM

**Review by SuperAayush — APPROVED** [review]
LGTM

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by SuperAayush — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by AayushShuk1a — APPROVED** [review]

**Review by AayushShuk1a — APPROVED** [review]
LGTM👍

File: server/Routes/Routes.js (hunk 1 above)
Comment (chirag0002) [reply server/Routes/Routes.js:30]: Removed all unnecessary consoles.

**Review by bhavyastar — COMMENTED** [review]
LGTM

File: server/Routes/Routes.js (hunk 2 above)
Comment (chirag0002) [reply server/Routes/Routes.js:57]: Comments are added to the codes where needed.

**Pull request merged**